src/components/AdminSettings/SignalingServer.vue

```diff
@@ -38,17 +38,23 @@
 			@change="updateVerify">
 		<label :for="'verify' + index">{{ t('spreed', 'Validate SSL certificate') }}</label>
 
-		<a v-show="!loading"
-			v-tooltip.auto="t('spreed', 'Delete this server')"
-			class="icon icon-delete"
-			@click="removeServer" />
+		<Button v-show="!loading"
+			type="tertiary-no-background"
+			:aria-label="t('spreed', 'Delete this server')"
+			@click="removeServer">
+			<template #icon>
+				<Delete />
```

Comment:
Unless you really want the icon to be the default size (`24`), you should probably set the size we agreed on kind of globally, which is `20`:
```suggestion
				<Delete :size="20"/>
```
see e.g. https://github.com/nextcloud/nextcloud-vue/blob/71468d3a42bc0e53899e31e1df6fdfb093ec3840/src/components/AppNavigationToggle/AppNavigationToggle.vue#L33

Reply:
Right, also fixed stun and turn icons